core/src/scheduled_jobs/state_dump.rs

```diff
@@ -38,35 +41,60 @@ fn address_to_content_and_type(address: &Address, context: Arc<Context>) -> Resu
 pub fn state_dump(context: Arc<Context>) {
     let (nucleus, network, dht) = {
         let state_lock = context.state().expect("No state?!");
-        ((*state_lock.nucleus()).clone(), (*state_lock.network()).clone(), (*state_lock.dht()).clone())
+        (
+            (*state_lock.nucleus()).clone(),
+            (*state_lock.network()).clone(),
+            (*state_lock.dht()).clone(),
+        )
     };
 
-
-    let running_calls: Vec<ZomeFnCall> = nucleus.zome_calls
+    let running_calls: Vec<ZomeFnCall> = nucleus
+        .zome_calls
         .into_iter()
         .filter(|(_, result)| result.is_none())
         .map(|(call, _)| call)
         .collect();
 
-    let get_entry_flows: Vec<Address> = network.get_entry_with_meta_results
-        .into_iter()
+    let get_entry_flows: Vec<Address> = network
+        .get_results
+        //using iter so that we don't copy this again and again if it is a scheduled job that runs everytime
+        //it might be slow if copied
+        .iter()
         .filter(|(_, result)| result.is_none())
-        .map(|(key, _)| key.address.clone())
+        .filter(|(key, _)| match key {
+            QueryKey::Entry(_) => true,
+            _ => false,
+        })
+        //this just ensures that we clone what we need
+        .map(|(key, _)| match key {
+            QueryKey::Entry(key) => key.address.clone(),
+            _ => panic!("Wrong Enum Variant should never be reached"),
+        })
         .collect();
 
-    let get_links_flows: Vec<GetLinksKey> = network.get_links_results
-        .into_iter()
+    let get_links_flows: Vec<GetLinksKey> = network
+        .get_results
+        .iter()
         .filter(|(_, result)| result.is_none())
-        .map(|(key, _)| key)
+        .filter(|(key, _)| match key {
+            QueryKey::Links(_) => true,
+            _ => false,
+        })
+        .map(|(key, _)| match key {
+            QueryKey::Links(key) => key.clone(),
+            _ => panic!("Wrong Enum Variant should never be reached"),
+        })
```

Comment:
Ah, it actually doesn't really make sense to keep these artificially separated here just to keep the state dump output the same. I think we should rather change the dump to not show separate get_entry/get_link_flows, but instead just running queries...